package.json

```diff
@@ -74,18 +74,18 @@
   "devDependencies": {
     "@babel/core": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
-    "babel-jest": "^24.8.0",
+    "babel-jest": "^25.3.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-transform-builtin-extend": "^1.1.2",
     "documentation": "^12.1.3",
     "dotenv": "^8.0.0",
     "eslint": "^6.6.0",
-    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-airbnb-base": "^14.1.0",
     "eslint-plugin-import": "^2.13.0",
-    "jest": "^24.8.0",
-    "jest-config": "^24.8.0",
+    "jest": "^25.3.0",
+    "jest-config": "^25.3.0",
     "klaw-sync": "^6.0.0",
-    "lint-staged": "^9.2.0",
+    "lint-staged": "^10.1.0",
```

Comment:
With `lint-staged` 10.x, I think we need to add husky (`npm install --save-dev husky`) after a fresh clone of the repo to make sure the older pre-commit hooks no longer exist.
Then add:
```json
    "husky": {
        "hooks": {
            "pre-commit": "lint-staged"
        }
    },
```
and remove `    "precommit": "lint-staged",`.